tests/Rector/ClassMethod/RemoveUnusedRequestParamRector/Fixture/controller.php.inc

```diff
@@ -0,0 +1,55 @@
+<?php
```

Comment:
please rename fixture file to something like: `do_not_override_parent_abstract_class_method.php.inc`

Reply:
++